src/target/semihosting.c

```diff
@@ -870,7 +870,7 @@ int32_t semihosting_request(target_s *const target, const uint32_t syscall, cons
 	target->tc->interrupted = false;
 
 	/* Set up the request block appropriately */
-	semihosting_s request = {r1, {}};
+	semihosting_s request = {r1, {0, 0, 0, 0}};
```

Comment:
Same comment as riscv_debug.c. The alternative patch (that works for us too) is:
```suggestion
	semihosting_s request = {r1};
```